### PR TITLE
added possibility to generate jacobian for specific link

### PR DESCRIPTION
**Pull request by claudius-kienle**
Kinpy currently only supports to compute the jacobian for the whole kinematics chain.
This pull request adds a new parameter `end_only` to the `jacobian` method in the class `SerialChain`,
that allows to generate the jacobian for all links.
The interface is inspired from the `forward` and therefore returns a dictionary with the jacobians for all links.

### CODE REVIEW

File: kinpy/jacobian.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from os import link
```
Comment (neka-nat) [inline kinpy/jacobian.py:1]: This is not used, is it?

File: kinpy/chain.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from os import link
```
Comment (neka-nat) [inline kinpy/chain.py:1]: This is not used, is it?

**Review by neka-nat — COMMENTED** [review]
Thanks for the pull request!
I have commented on two points.

File: kinpy/jacobian.py (hunk 1 above)
Comment (claudius-kienle) [reply kinpy/jacobian.py]: Year, you're right. I removed this import.

File: kinpy/chain.py (hunk 2 above)
Comment (claudius-kienle) [reply kinpy/chain.py]: Year, you're right. I removed this import.

**Review by neka-nat — APPROVED** [review]

**Pull request merged**